recipes/kim_convergence/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "kim-convergence" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1fb51b91e860581e3204c71bf4616786a25b2ebe6ac097020437a823f2b66505 
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - kim-edn
+    - numpy >=1.16
+    - scipy
+    - joblib
+
+test:
+  imports:
+    - kim_edn
+    - numpy
+    - scipy
+    - joblib
+    - kim_convergence
```

You should only test importing the package itself (and any subpackages), not its dependencies:
```suggestion
    - kim_convergence
```

---

@dopplershift thanks, done.